### PR TITLE
Fix the error messages when a templating error occurs and log them in the WARNING level

### PR DESCRIPTION
**Pull request by mertyildiran**

### CODE REVIEW

File: mockintosh/hbs/methods.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def reg_ex(this, regex, *args, **kwargs):
 
 def json_path(this, text, path):
     data = text
-    if data is None:
+    if isinstance(data, tuple) and data[0] is None:
```
Comment (undera) [inline mockintosh/hbs/methods.py:49]: Why do we need this check? `null` is a valid JSON document and `$` as JSONPath would succesfully match its root element.

File: requirements.txt [hunk 2]
```diff
@@ -3,6 +3,6 @@ Faker==4.18.0
 tornado==6.0.4
 PyYAML==5.3.1
 jsonschema==3.2.0
-pybars4>=0.9.8
+pybars4 @ git+https://github.com/up9inc/pybars4@3f294fc32e9c9034a73022cf37c7cee070338b04
```
Comment (undera) [inline requirements.txt:6]: PyPi does now allow this, it makes mockintosh not installable

File: mockintosh/hbs/methods.py (hunk 1 above)
Comment (mertyildiran) [reply mockintosh/hbs/methods.py:49]: This was related to the change that I've made in here https://github.com/up9inc/pybars4/commit/3f294fc32e9c9034a73022cf37c7cee070338b04#diff-e3fbc2a2ce5876feb7c919471cdb1cefea61644a6e7ba15d61c2571ab71b5aa8R325

File: requirements.txt (hunk 2 above)
Comment (mertyildiran) [reply requirements.txt:6]: This is for demonstrating the current state and making the tests pass. I was going to change it to something like `pybars4>=0.9.9` in the end.

**Pull request merged**